lessons/cryptography/cryptography-hashing-checksums-salts.md

```diff
@@ -0,0 +1,82 @@
+# [Cryptography] Hashing, Checksums, and Salts
+*What is data integrity and how is it verified?*
+
+![banner alt text](.rsrc/banner.png)
+
+# Objectives
```

Comment:
These are good objectives but there's a lot here. I think we should either reduce this or if the lesson is just really dense right now maybe we cut this into a few different ones?